### PR TITLE
Bump ClimaCore to 0.11.0

### PR DESCRIPTION
**Pull request by Sbozzolo**
New minor version because it breaks some of the subpackages

### CODE REVIEW

**Comment by sriharshakandala** [comment]
Can we also update dependencies. Otherwise looks good to me!

**Review by sriharshakandala — APPROVED** [review]

File: lib/ClimaCoreMakie/Project.toml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 authors = ["CliMA Contributors <clima-software@caltech.edu>"]
 name = "ClimaCoreMakie"
 uuid = "908f55d8-4145-4867-9c14-5dad1a479e4d"
-version = "0.4.0"
+version = "0.4.1"
 
 [compat]
-ClimaCore = "0.7, 0.8, 0.9, 0.10"
+ClimaCore = "0.7, 0.8, 0.9, 0.10, 0.11"
```
Comment (simonbyrne) [inline lib/ClimaCoreMakie/Project.toml:7]: I actually don't think these subpackages are all backward compatible.

Comment (Sbozzolo) [reply lib/ClimaCoreMakie/Project.toml:7]: I'll just keep `0.11` then.

**Comment by sriharshakandala** [comment]
We may not need `UnPack` anymore!

**Comment by Sbozzolo** [comment]
> We may not need `UnPack` anymore!

Do you want to add a commit to this PR that removes it?

**Comment by Sbozzolo** [comment]
> > We may not need `UnPack` anymore!
> 
> Do you want to add a commit to this PR that removes it?

Removed!

**Pull request merged**